_pages/install/browser-support.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Web Browser Support
+---
+
+Ideally, your storefront will render uniformly in any browser that is used by your customers. However, web browsers do not all support the same web standards, nor do they support the same standards in the same way. For example, there are different ways of interpreting a standard's specifications, deliberate design decisions that result in different outcomes, as well as bugs that are unique to each browser. And for each browser that is supported, there is a resulting implementation effort that includes QA, architectural decisions, and technical decisions. In the end, a balance has to be struck between providing an excellent storefront experience, and supporting (as much as possible) the different web browsers that are out there.
+
+Spartacus libraries are designed to take advantage of the latest web platform standards, while also allowing you to run your Spartacus storefront in as many different web browsers as possible. However, some older browsers do not support the latest standards, and as a result, these browsers are not supported by Spartacus. Spartacus actively supports "evergreen browsers", which means there are end-to-end tests and manual QA done for evergreen browsers (though other browsers that are not evergreen may be supported "by accident" - for example, if they support the latest web platform standards). Evergreen browsers are web browsers that are automatically upgraded to future versions, rather than being updated by distribution of a new version (for example, in an OS update). Spartacus actively supports only evergreen browsers because Spartacus follows standard Angular, the Spartacus libraries are kept as clean as possible by intentionally avoiding "browser quirks" in the library code, and there is no planned investment in automated tests of very old browsers.
+
+The approach to browser support in Spartacus is as follows:
+
+- Angular is fundamental for compatibility with evergreen browsers and beyond (for example IE11).
+- Other standards are leveraged to provide compatibility with evergreen browsers ([caniuse.com](http://caniuse.com/) is a good example).
+- Automated quality checks align the codebase with those standards (such as the TypeScript standard, and [caniuse.com](http://caniuse.com/)).
+- End-to-end testing is used to test the experience (only done in the Chrome browser).
+
+Nonetheless, with a bit of effort, it is still possible to get Spartacus to work with older browsers. The following are some of the steps you can take to get an older browsers to work with Spartacus:
```

Comment:
With dropping the previous section, I imagine we change the prelude a bit.  

While Spartacus might does not support older browsers, there are some common techniques to support older browsers. The following are some of the steps you can take to get an older browsers to work with Spartacus: